mobile_verifier/src/coverage.rs

```diff
@@ -402,47 +456,72 @@ impl CoveredHexes {
             signal_level,
             coverage_claim_time,
             radio_key,
+            signal_power,
             ..
         }) = covered_hexes.next().await.transpose()?
         {
-            self.hexes
-                .entry(CellIndex::try_from(hex as u64).unwrap())
-                .or_default()[indoor as usize]
-                .entry(signal_level)
-                .or_default()
-                .push(CoverageLevel {
-                    radio_key,
-                    coverage_claim_time,
-                    indoor,
-                    signal_level,
-                    hotspot: hotspot.clone(),
-                });
+            if indoor {
+                self.indoor
+                    .entry(CellIndex::try_from(hex as u64).unwrap())
```

Comment:
I don't know about h3o vs heron, but we instantiate the cell index using the same method before putting it in the DB

Reply:
i guess the question then wouldn't really apply here where you're reading out of the DB but rather at the insert point; if the u64 being turned into a CellIndex there isn't a valid H3 cell h3o will perform that validation and fail, which would cause the loading process to crash whereas working with heron previously the library would allow you to instantiate invalid cells